app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+//= require_tree .
+//= require_self
+
+@import url("http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,400,300,600");
+
+body {
+  margin: 60px;
+  font-family: 'Open Sans', sans-serif;
+  p {
+    font-size: 1em;
+    font-weight: 300;
+  }
+  h1 {
+    font-size: 2.4em;
+    font-weight: 600;
+  }
+  h2 {
+    font-size: 2em;
+    font-weight: 600;
+  }
+  h3 {
+    font-size: 1.6em;
+    font-weight: 600;
+  }
+  h4 {
+    font-size: 1.2em;
+    font-weight: 600;
+  }
+  a {
+    color: #0066FF;
```

Comment:
Color literals like `#0066FF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#0066FF` should be written as `#0066ff`<br>Properties should be ordered border-bottom, color, font-size, font-weight, text-decoration